### ISSUE TITLE
Batchsize and lr of HTC+Swin-L

### ISSUE THREAD

**Issue by BinZhu-ece** [VDIGPKU/CBNetV2#50]
Thank you for your outstanding work, I only have 8 A-100 cards, can I train a batchsize each card and adjust the learning rate to 0.0001 / 2? Is there a negative impact on this relatively large model accuracy of HTC+swin-L?

**Comment by mitming** [comment]
> Thank you for your outstanding work, I only have 8 A-100 cards, can I train a batchsize each card and adjust the learning rate to 0.0001 / 2? Is there a negative impact on this relatively large model accuracy of HTC+swin-L?

Hi, @962858249 

Could you run the code successfully? There are a lot of bugs when I tried to run the config of `HTC + Swin-L`

**Comment by clw5180** [comment]
> Thank you for your outstanding work, I only have 8 A-100 cards, can I train a batchsize each card and adjust the learning rate to 0.0001 / 2? Is there a negative impact on this relatively large model accuracy of HTC+swin-L?

only用得好
